content/tutorials/r_setup_codingclub_session/index.Rmd

```diff
@@ -0,0 +1,75 @@
+---
+title: "Tutorial setup_codingclub_session()"
+description: "Tutorial on how to use setup_codingclub_session() to load data and scripts to use during coding clubs."
+author: "Emma Cartuyvels"
+date: 2020-10-26
+categories: ["r"]
+tags: ["data", "r"]
+output: 
+    md_document:
+        preserve_yaml: true
+        variant: gfm
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(eval = FALSE)
+```
+
+
+Before each coding club you will usually have to download some data and some scripts.
+This can be done manually by going to each of the GitHub pages but it is much easier to use the `setup_codingclub_session()` function of the `inborutils` package.
+
+To begin make sure that you have the latest version of `inborutils` installed by running the following code:
+
+```{r}
+if (!"remotes" %in% rownames(installed.packages())) {
+  install.packages("remotes")
+}
+remotes::install_github("inbo/inborutils")
+```
+
+Next load the library:
+
+```{r}
+library(inborutils)
+```
+
+The function takes four arguments with the following default settings:
```

Comment:
Before a list of items, a whiteline is needed. (In index.md it is wrapped into one paragraph)
```suggestion
The function takes four arguments with the following default settings:

```